components/TopBarProfileMenu.js

```diff
@@ -329,6 +336,28 @@ class TopBarProfileMenu extends React.Component {
                 </P>
               </Box>
             )}
+            {events.length > 0 && (
+              <div>
+                <Flex alignItems="center">
```

Comment:
To match the other margins:

```suggestion
                <Flex alignItems="center" mt={3}>
```